server/src/graphql-types/TokenStatus.ts

```diff
@@ -0,0 +1,10 @@
+import { ObjectType, Field } from 'type-graphql';
+
+@ObjectType()
+export class TokenStatus {
+  @Field(() => String)
+  email: string;
```

Comment:
```suggestion
  redacted_email: string;
```
This might be overkill, but it makes it abundantly clear what this is.

Reply:
I like explicit :rocket: